llvm/docs/CommandGuide/llvm-dwarfdump.rst

```diff
@@ -45,6 +45,16 @@ OPTIONS
 
             Use colors in output.
 
+.. option:: --error-display=<value>       
+
+            Set the level of detail and summary to display when verifying.
+            Implies :option:`--verify`. The supported values are:
+
+            `quiet`   - Only display whether errors occurred.
+            `summary` - Display only a summary of the errors found.
+            `details` - Display each error in detail but no summary.
+            `full`    - Display each error as well as a summary. [default]
```

Comment:
This list isn't formatted correctly and doesn't display like you probably intended. See https://llvm.org/docs/CommandGuide/llvm-dwarfdump.html#cmdoption-llvm-dwarfdump-error-display

Reply:
Thanks. Doesn't look too bad actually but definitely not what was intended. Fix: https://github.com/llvm/llvm-project/pull/111693.